src/sim.rs

```diff
@@ -1062,12 +1062,12 @@ impl Simulation {
                                 );
                         }
                     } else {
-                        0.5 // default to centering
+                        0.0 // default to soma location
```

Comment:
I think, since we rely on this quite heavily, in the absence of (x, y, z) an error should be raised.